node/node_test.go

```diff
@@ -359,6 +360,10 @@ func TestCommit(t *testing.T) {
 
 	cn := newTestNode(ctx, mn, t)
 
+	// we set an upper bound limit lower, so that when we add the 2nd file
+	// it will force an eviction
+	cn.exch.Index().UpdateOptions(exchange.WithBounds(255000, 128000))
```

Comment:
This is where I change the upper bound limit, so that when adding a new file, the GC will evict the first file

Reply:
Why not just set the capacity in the options? This seems confusing and like adding more complexity.

Reply:
I tried but on a node is created there is no way to change the Capacity, but you're right, I'll find a way to add custom params to a node, in the test, without adding complexity to the codebase.